lib/runner/cli/cli.js

```diff
@@ -392,11 +392,10 @@ class CliRunner {
   isTestWorkersEnabled() {
     const testWorkers =  this.test_settings.testWorkersEnabled && !singleSourceFile(this.argv);
     if (!testWorkers) {
-      return false;
-    }
 
-    if (this.argv.debug) {
-      Logger.info('Disabling parallelism while running in debug mode');
+      if (this.argv.debug) {
+        Logger.info('Disabling parallelism while running in debug mode');
+      }
```

Comment:
The only downside to this would. We don't really know if the user did pass `--parallel` along with `--debug` and we would print this anyway if the test workers are found disabled.

Reply:
Yes I thought of that, but assuming parallel runs are enabled by default, I think this is acceptable. Does not mislead the user in my opinion. And keeps the implementation simple.